### PR TITLE
[skip ci] Made post-installation messages more legible in install.bat

### LINKED ISSUE #2672: Successful UI client installation message is a bit misleading
When you successfully deploy or upgrade the Web client plug-in for VIC Engine, you see the following message:

ui_install

The presence of the big message about copying the files to VC initially made me think that the deployment hadn't worked. Could we make the message a little clearer that the installation succeeded?

For example:

```
Installing plugin
Installation of UI plugin succeeded

====
NOTE: With the current version of VIC, if you have not done so already, you have to manually copy ....
etc.
====
```

By adding the words `succeeded`, `NOTE`, and `if you have not done so already`, I think that this makes it clearer that the deployment worked. @jooskim, what do you think?

### PR DESCRIPTION
**Pull request by jooskim**
Post-installation messages in the VIC plugin installer script for vCenter for Windows were updated to be more legible. Fixes #2672

### CODE REVIEW

File: ui/installer/vCenterForWindows/install.bat [hunk 1]
```diff
@@ -79,15 +79,18 @@ FOR /F "tokens=*" %%A IN (..\vCenterForWindows\_scratch_flags.txt) DO (
...
-    ECHO With the current version of VIC, you have to manually copy the com.vmware.vicui.* folder in \ui\vsphere-client-serenity to %VMWARE_CFG_DIR%\vsphere-client\vc-packages\vsphere-client-serenity
-    ECHO
-    ECHO Note for vCenter 5.5 users:
-    ECHO VIC UI may run on a vCenter 5.5 setup, but is not officially supported. Use it with your own risk. To proceed, copy the aforementioned folder to %PROGRAMDATA%\VMware\vSphere Web Client\vc-packages\vsphere-client-serenity instead
-    ECHO
+    ECHO ** NEXT STEP for vCenter 6.0 users **
+    ECHO With the current version of VIC running on vCenter for Windows, the com.vmware.vicui.* folder needs to be manually copied from \ui\vsphere-client-serenity to %VMWARE_CFG_DIR%\vsphere-client\vc-packages\vsphere-client-serenity. If you have not done so, please copy it now.
+    ECHO.
+    ECHO ** NEXT STEP for vCenter 5.5 users **
+    ECHO VIC UI may run on a vCenter 5.5 setup, but is NOT officially supported. Use it with your own risk. To proceed, copy the com.vmware.vicui.* folder to %PROGRAMDATA%\VMware\vSphere Web Client\vc-packages\vsphere-client-serenity instead.
```
Comment (andrewtchin) [inline ui/installer/vCenterForWindows/install.bat]: `Use it at your own risk`

**Comment by andrewtchin** [comment]
lgtm

**Comment by hickeng** [comment]
lgtm

File: ui/installer/vCenterForWindows/install.bat (hunk 1 above)
Comment (jooskim) [inline ui/installer/vCenterForWindows/install.bat]: ha. i feel stupid now for making that mistake. thanks anyway.

**Comment by jooskim** [comment]
🔵

**Comment by stuclem** [comment]
Thanks @jooskim!

**Pull request merged**